packages/templates/clients/websocket/python/example.py

```diff
@@ -1,4 +1,10 @@
+import sys
+import os
+
+sys.path.append(os.path.join(os.path.dirname(__file__), 'test'))
```

Comment:
why this? wasn't it enough to update below by adding `test` like `test.temp.snapshotTestResult.client_postman`?

Reply:
yeah, let is leave it like this, 

other solution would be to put empty `__init__.py` in test folder, but in case of our node project I think it is worse than what we have now in the example